.github/container/test-t5x.sh

```diff
@@ -7,7 +7,7 @@ print_var() {
 }
 
 usage() {
-    echo "Test T5X throughput on a fake-data Wikipedia benchmark."
+    echo "Test T5X throughput on a fake-data Wikipedia benchmark or real squad."
```

Comment:
What is 'real squad'?

Reply:
By real squad, I mean "Stanford Question Answering Dataset (SQuAD)". I can use this casing to avoid confusion